class-pinterest-for-woocommerce.php

```diff
@@ -709,21 +709,10 @@ public static function update_account_data() {
 		 */
 		public static function get_linked_businesses( $force_refresh = false ) {
 
-			$linked_businesses = Pinterest_For_Woocommerce()::get_data( 'linked_businesses' );
-
-			if ( $force_refresh || is_null( $linked_businesses ) ) {
-
-				$account_data       = Pinterest_For_Woocommerce()::get_setting( 'account_data' );
-				$fetched_businesses = ( ! empty( $account_data ) && ! $account_data['is_partner'] ) ? Pinterest\API\Base::get_linked_businesses() : array();
-
-				if ( ! empty( $fetched_businesses ) && 'success' === $fetched_businesses['status'] ) {
-					$linked_businesses = $fetched_businesses['data'];
-				}
-
-				$linked_businesses = $linked_businesses ?? array();
-
-				self::save_data( 'linked_businesses', $linked_businesses );
+			$linked_businesses = ! $force_refresh ? Pinterest_For_Woocommerce()::get_data( 'linked_businesses' ) : null;
 
+			if ( null === $linked_businesses ) {
+				$linked_businesses = self::update_linked_businesses( $linked_businesses );
```

Comment:
This method doesn't require an argument.

```suggestion
				$linked_businesses = self::update_linked_businesses();
```